.env.example

```diff
@@ -0,0 +1,2 @@
+# RENAME TO .env AND ASK TEAM MEMBER FOR VALUES
```

Comment:
Can we make this `COPY` instead of `RENAME`?

Reply:
Done